SystemDynamics/package.mo

```diff
@@ -1,4 +1,4 @@
-package SystemDynamics "System Dynamics Library (Version 2.1.1)"
+﻿package SystemDynamics "System Dynamics Library (Version 2.1.1)"
```

Comment:
Here still the old version is present. But it is also a strange place to have the version number. So maybe just remove the version from the package description here.

Reply:
Ah, I missed that one. I will remove it as is not needed, the tools display the version.